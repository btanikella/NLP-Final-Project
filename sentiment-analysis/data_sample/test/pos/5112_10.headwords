movie
were
funny
Visit
recommend
try
are
So
find
