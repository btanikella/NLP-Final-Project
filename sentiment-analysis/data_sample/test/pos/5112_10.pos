DT VBZ DT JJ NN
EX VBD JJ NNS TO DT NN PRP VBD
DT NN VBZ RB JJ
VB DT NNP NN NN NN NN IN NNP EX VBP JJ NNS JJ IN DT NN : NNP VBZ TO NNP  NNP VBZ NNP  NNP VBZ TO NN  NNP IN DT NNP  NNP VBZ TO NNP  NNP NNP NNP NNP NNP NNP FW
PRP RB VB DT IN NN NNS
DT NN NNP NNP RB VBP VBG NNP WDT VBZ DT JJS NNP NN
IN PRP$ NN EX VBP RB CD NNP VBZ TO NNP CC NNP VBZ TO NN
RB IN PRP VBP RB VBN NNP NNP NNP PRP$ NN TO VB CC VB PRP
PRP MD VB PRP RB JJ  DT WP PRP VBP
