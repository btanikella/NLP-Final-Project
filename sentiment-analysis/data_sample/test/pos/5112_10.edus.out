Sentence #1 (6 tokens):
This is an excellent movie.
[Text=This CharacterOffsetBegin=4 CharacterOffsetEnd=8 PartOfSpeech=DT] [Text=is CharacterOffsetBegin=9 CharacterOffsetEnd=11 PartOfSpeech=VBZ] [Text=an CharacterOffsetBegin=12 CharacterOffsetEnd=14 PartOfSpeech=DT] [Text=excellent CharacterOffsetBegin=15 CharacterOffsetEnd=24 PartOfSpeech=JJ] [Text=movie CharacterOffsetBegin=25 CharacterOffsetEnd=30 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=30 CharacterOffsetEnd=31 PartOfSpeech=.] 
root(ROOT-0, movie-5)
nsubj(movie-5, This-1)
cop(movie-5, is-2)
det(movie-5, an-3)
amod(movie-5, excellent-4)
punct(movie-5, .-6)

Sentence #2 (10 tokens):
There were several parts to the movie I liked.
[Text=There CharacterOffsetBegin=36 CharacterOffsetEnd=41 PartOfSpeech=EX] [Text=were CharacterOffsetBegin=42 CharacterOffsetEnd=46 PartOfSpeech=VBD] [Text=several CharacterOffsetBegin=47 CharacterOffsetEnd=54 PartOfSpeech=JJ] [Text=parts CharacterOffsetBegin=55 CharacterOffsetEnd=60 PartOfSpeech=NNS] [Text=to CharacterOffsetBegin=61 CharacterOffsetEnd=63 PartOfSpeech=TO] [Text=the CharacterOffsetBegin=64 CharacterOffsetEnd=67 PartOfSpeech=DT] [Text=movie CharacterOffsetBegin=68 CharacterOffsetEnd=73 PartOfSpeech=NN] [Text=I CharacterOffsetBegin=74 CharacterOffsetEnd=75 PartOfSpeech=PRP] [Text=liked CharacterOffsetBegin=76 CharacterOffsetEnd=81 PartOfSpeech=VBD] [Text=. CharacterOffsetBegin=81 CharacterOffsetEnd=82 PartOfSpeech=.] 
root(ROOT-0, were-2)
expl(were-2, There-1)
amod(parts-4, several-3)
nsubj(were-2, parts-4)
prep(were-2, to-5)
det(movie-7, the-6)
pobj(to-5, movie-7)
nsubj(liked-9, I-8)
rcmod(movie-7, liked-9)
punct(were-2, .-10)

Sentence #3 (6 tokens):
This movie is very funny!
[Text=This CharacterOffsetBegin=87 CharacterOffsetEnd=91 PartOfSpeech=DT] [Text=movie CharacterOffsetBegin=92 CharacterOffsetEnd=97 PartOfSpeech=NN] [Text=is CharacterOffsetBegin=98 CharacterOffsetEnd=100 PartOfSpeech=VBZ] [Text=very CharacterOffsetBegin=101 CharacterOffsetEnd=105 PartOfSpeech=RB] [Text=funny CharacterOffsetBegin=106 CharacterOffsetEnd=111 PartOfSpeech=JJ] [Text=! CharacterOffsetBegin=111 CharacterOffsetEnd=112 PartOfSpeech=.] 
root(ROOT-0, funny-5)
det(movie-2, This-1)
nsubj(funny-5, movie-2)
cop(funny-5, is-3)
advmod(funny-5, very-4)
punct(funny-5, !-6)

Sentence #4 (50 tokens):
Visit the Ernest fun club web site at www.ernestfunclub.com There are several movies such as the following: Ernest Goes to Camp, Ernest Saves Christmas, Ernest Goes to Jail, Ernest in the Army, Ernest Goes to School, Ernest Rides Again Slam Dunk Ernest etc.
[Text=Visit CharacterOffsetBegin=117 CharacterOffsetEnd=122 PartOfSpeech=VB] [Text=the CharacterOffsetBegin=123 CharacterOffsetEnd=126 PartOfSpeech=DT] [Text=Ernest CharacterOffsetBegin=127 CharacterOffsetEnd=133 PartOfSpeech=NNP] [Text=fun CharacterOffsetBegin=134 CharacterOffsetEnd=137 PartOfSpeech=NN] [Text=club CharacterOffsetBegin=138 CharacterOffsetEnd=142 PartOfSpeech=NN] [Text=web CharacterOffsetBegin=143 CharacterOffsetEnd=146 PartOfSpeech=NN] [Text=site CharacterOffsetBegin=147 CharacterOffsetEnd=151 PartOfSpeech=NN] [Text=at CharacterOffsetBegin=152 CharacterOffsetEnd=154 PartOfSpeech=IN] [Text=www.ernestfunclub.com CharacterOffsetBegin=155 CharacterOffsetEnd=176 PartOfSpeech=NNP] [Text=There CharacterOffsetBegin=177 CharacterOffsetEnd=182 PartOfSpeech=EX] [Text=are CharacterOffsetBegin=183 CharacterOffsetEnd=186 PartOfSpeech=VBP] [Text=several CharacterOffsetBegin=187 CharacterOffsetEnd=194 PartOfSpeech=JJ] [Text=movies CharacterOffsetBegin=195 CharacterOffsetEnd=201 PartOfSpeech=NNS] [Text=such CharacterOffsetBegin=202 CharacterOffsetEnd=206 PartOfSpeech=JJ] [Text=as CharacterOffsetBegin=207 CharacterOffsetEnd=209 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=210 CharacterOffsetEnd=213 PartOfSpeech=DT] [Text=following CharacterOffsetBegin=214 CharacterOffsetEnd=223 PartOfSpeech=NN] [Text=: CharacterOffsetBegin=223 CharacterOffsetEnd=224 PartOfSpeech=:] [Text=Ernest CharacterOffsetBegin=225 CharacterOffsetEnd=231 PartOfSpeech=NNP] [Text=Goes CharacterOffsetBegin=232 CharacterOffsetEnd=236 PartOfSpeech=VBZ] [Text=to CharacterOffsetBegin=237 CharacterOffsetEnd=239 PartOfSpeech=TO] [Text=Camp CharacterOffsetBegin=240 CharacterOffsetEnd=244 PartOfSpeech=NNP] [Text=, CharacterOffsetBegin=244 CharacterOffsetEnd=245 PartOfSpeech=,] [Text=Ernest CharacterOffsetBegin=246 CharacterOffsetEnd=252 PartOfSpeech=NNP] [Text=Saves CharacterOffsetBegin=253 CharacterOffsetEnd=258 PartOfSpeech=VBZ] [Text=Christmas CharacterOffsetBegin=259 CharacterOffsetEnd=268 PartOfSpeech=NNP] [Text=, CharacterOffsetBegin=268 CharacterOffsetEnd=269 PartOfSpeech=,] [Text=Ernest CharacterOffsetBegin=270 CharacterOffsetEnd=276 PartOfSpeech=NNP] [Text=Goes CharacterOffsetBegin=277 CharacterOffsetEnd=281 PartOfSpeech=VBZ] [Text=to CharacterOffsetBegin=282 CharacterOffsetEnd=284 PartOfSpeech=TO] [Text=Jail CharacterOffsetBegin=285 CharacterOffsetEnd=289 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=289 CharacterOffsetEnd=290 PartOfSpeech=,] [Text=Ernest CharacterOffsetBegin=291 CharacterOffsetEnd=297 PartOfSpeech=NNP] [Text=in CharacterOffsetBegin=298 CharacterOffsetEnd=300 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=301 CharacterOffsetEnd=304 PartOfSpeech=DT] [Text=Army CharacterOffsetBegin=305 CharacterOffsetEnd=309 PartOfSpeech=NNP] [Text=, CharacterOffsetBegin=309 CharacterOffsetEnd=310 PartOfSpeech=,] [Text=Ernest CharacterOffsetBegin=311 CharacterOffsetEnd=317 PartOfSpeech=NNP] [Text=Goes CharacterOffsetBegin=318 CharacterOffsetEnd=322 PartOfSpeech=VBZ] [Text=to CharacterOffsetBegin=323 CharacterOffsetEnd=325 PartOfSpeech=TO] [Text=School CharacterOffsetBegin=326 CharacterOffsetEnd=332 PartOfSpeech=NNP] [Text=, CharacterOffsetBegin=332 CharacterOffsetEnd=333 PartOfSpeech=,] [Text=Ernest CharacterOffsetBegin=334 CharacterOffsetEnd=340 PartOfSpeech=NNP] [Text=Rides CharacterOffsetBegin=341 CharacterOffsetEnd=346 PartOfSpeech=NNP] [Text=Again CharacterOffsetBegin=347 CharacterOffsetEnd=352 PartOfSpeech=NNP] [Text=Slam CharacterOffsetBegin=353 CharacterOffsetEnd=357 PartOfSpeech=NNP] [Text=Dunk CharacterOffsetBegin=358 CharacterOffsetEnd=362 PartOfSpeech=NNP] [Text=Ernest CharacterOffsetBegin=363 CharacterOffsetEnd=369 PartOfSpeech=NNP] [Text=etc. CharacterOffsetBegin=370 CharacterOffsetEnd=373 PartOfSpeech=FW] [Text=. CharacterOffsetBegin=373 CharacterOffsetEnd=374 PartOfSpeech=.] 
root(ROOT-0, Visit-1)
det(site-7, the-2)
nn(site-7, Ernest-3)
nn(site-7, fun-4)
nn(site-7, club-5)
nn(site-7, web-6)
dobj(Visit-1, site-7)
prep(site-7, at-8)
pobj(at-8, www.ernestfunclub.com-9)
expl(are-11, There-10)
dep(Visit-1, are-11)
amod(movies-13, several-12)
nsubj(are-11, movies-13)
mwe(as-15, such-14)
prep(movies-13, as-15)
det(following-17, the-16)
pobj(as-15, following-17)
punct(are-11, :-18)
nsubj(Goes-20, Ernest-19)
parataxis(are-11, Goes-20)
prep(Goes-20, to-21)
pobj(to-21, Camp-22)
punct(Goes-20, ,-23)
nn(Christmas-26, Ernest-24)
nn(Christmas-26, Saves-25)
dep(Goes-20, Christmas-26)
punct(Christmas-26, ,-27)
nn(Goes-29, Ernest-28)
conj(Christmas-26, Goes-29)
prep(Goes-29, to-30)
pobj(to-30, Jail-31)
punct(Christmas-26, ,-32)
conj(Christmas-26, Ernest-33)
prep(Ernest-33, in-34)
det(Army-36, the-35)
pobj(in-34, Army-36)
punct(Christmas-26, ,-37)
nn(Goes-39, Ernest-38)
conj(Christmas-26, Goes-39)
prep(Goes-39, to-40)
pobj(to-40, School-41)
punct(Christmas-26, ,-42)
nn(etc.-49, Ernest-43)
nn(etc.-49, Rides-44)
nn(etc.-49, Again-45)
nn(etc.-49, Slam-46)
nn(etc.-49, Dunk-47)
nn(etc.-49, Ernest-48)
appos(Christmas-26, etc.-49)
punct(Visit-1, .-50)

Sentence #5 (8 tokens):
I highly recommend these for family movies.
[Text=I CharacterOffsetBegin=379 CharacterOffsetEnd=380 PartOfSpeech=PRP] [Text=highly CharacterOffsetBegin=381 CharacterOffsetEnd=387 PartOfSpeech=RB] [Text=recommend CharacterOffsetBegin=388 CharacterOffsetEnd=397 PartOfSpeech=VB] [Text=these CharacterOffsetBegin=398 CharacterOffsetEnd=403 PartOfSpeech=DT] [Text=for CharacterOffsetBegin=404 CharacterOffsetEnd=407 PartOfSpeech=IN] [Text=family CharacterOffsetBegin=408 CharacterOffsetEnd=414 PartOfSpeech=NN] [Text=movies CharacterOffsetBegin=415 CharacterOffsetEnd=421 PartOfSpeech=NNS] [Text=. CharacterOffsetBegin=421 CharacterOffsetEnd=422 PartOfSpeech=.] 
root(ROOT-0, recommend-3)
nsubj(recommend-3, I-1)
advmod(recommend-3, highly-2)
dobj(recommend-3, these-4)
prep(these-4, for-5)
nn(movies-7, family-6)
pobj(for-5, movies-7)
punct(recommend-3, .-8)

Sentence #6 (15 tokens):
All star Jim Varney again try visiting www.ernestfunclub.com Which is the best Ernest movie?
[Text=All CharacterOffsetBegin=427 CharacterOffsetEnd=430 PartOfSpeech=DT] [Text=star CharacterOffsetBegin=431 CharacterOffsetEnd=435 PartOfSpeech=NN] [Text=Jim CharacterOffsetBegin=436 CharacterOffsetEnd=439 PartOfSpeech=NNP] [Text=Varney CharacterOffsetBegin=440 CharacterOffsetEnd=446 PartOfSpeech=NNP] [Text=again CharacterOffsetBegin=447 CharacterOffsetEnd=452 PartOfSpeech=RB] [Text=try CharacterOffsetBegin=453 CharacterOffsetEnd=456 PartOfSpeech=VBP] [Text=visiting CharacterOffsetBegin=457 CharacterOffsetEnd=465 PartOfSpeech=VBG] [Text=www.ernestfunclub.com CharacterOffsetBegin=466 CharacterOffsetEnd=487 PartOfSpeech=NNP] [Text=Which CharacterOffsetBegin=488 CharacterOffsetEnd=493 PartOfSpeech=WDT] [Text=is CharacterOffsetBegin=494 CharacterOffsetEnd=496 PartOfSpeech=VBZ] [Text=the CharacterOffsetBegin=497 CharacterOffsetEnd=500 PartOfSpeech=DT] [Text=best CharacterOffsetBegin=501 CharacterOffsetEnd=505 PartOfSpeech=JJS] [Text=Ernest CharacterOffsetBegin=506 CharacterOffsetEnd=512 PartOfSpeech=NNP] [Text=movie CharacterOffsetBegin=513 CharacterOffsetEnd=518 PartOfSpeech=NN] [Text=? CharacterOffsetBegin=518 CharacterOffsetEnd=519 PartOfSpeech=.] 
root(ROOT-0, try-6)
det(Varney-4, All-1)
nn(Varney-4, star-2)
nn(Varney-4, Jim-3)
nsubj(try-6, Varney-4)
advmod(try-6, again-5)
xcomp(try-6, visiting-7)
dobj(visiting-7, www.ernestfunclub.com-8)
nsubj(movie-14, Which-9)
cop(movie-14, is-10)
det(movie-14, the-11)
amod(movie-14, best-12)
nn(movie-14, Ernest-13)
rcmod(www.ernestfunclub.com-8, movie-14)
punct(try-6, ?-15)

Sentence #7 (17 tokens):
In my opinion there are actually 2 Ernest Goes to Camp and Ernest Goes to Jail.
[Text=In CharacterOffsetBegin=524 CharacterOffsetEnd=526 PartOfSpeech=IN] [Text=my CharacterOffsetBegin=527 CharacterOffsetEnd=529 PartOfSpeech=PRP$] [Text=opinion CharacterOffsetBegin=530 CharacterOffsetEnd=537 PartOfSpeech=NN] [Text=there CharacterOffsetBegin=538 CharacterOffsetEnd=543 PartOfSpeech=EX] [Text=are CharacterOffsetBegin=544 CharacterOffsetEnd=547 PartOfSpeech=VBP] [Text=actually CharacterOffsetBegin=548 CharacterOffsetEnd=556 PartOfSpeech=RB] [Text=2 CharacterOffsetBegin=557 CharacterOffsetEnd=558 PartOfSpeech=CD] [Text=Ernest CharacterOffsetBegin=559 CharacterOffsetEnd=565 PartOfSpeech=NNP] [Text=Goes CharacterOffsetBegin=566 CharacterOffsetEnd=570 PartOfSpeech=VBZ] [Text=to CharacterOffsetBegin=571 CharacterOffsetEnd=573 PartOfSpeech=TO] [Text=Camp CharacterOffsetBegin=574 CharacterOffsetEnd=578 PartOfSpeech=NNP] [Text=and CharacterOffsetBegin=579 CharacterOffsetEnd=582 PartOfSpeech=CC] [Text=Ernest CharacterOffsetBegin=583 CharacterOffsetEnd=589 PartOfSpeech=NNP] [Text=Goes CharacterOffsetBegin=590 CharacterOffsetEnd=594 PartOfSpeech=VBZ] [Text=to CharacterOffsetBegin=595 CharacterOffsetEnd=597 PartOfSpeech=TO] [Text=Jail CharacterOffsetBegin=598 CharacterOffsetEnd=602 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=602 CharacterOffsetEnd=603 PartOfSpeech=.] 
root(ROOT-0, are-5)
prep(are-5, In-1)
poss(opinion-3, my-2)
pobj(In-1, opinion-3)
expl(are-5, there-4)
advmod(Goes-9, actually-6)
num(Ernest-8, 2-7)
nsubj(Goes-9, Ernest-8)
nsubj(are-5, Goes-9)
prep(Goes-9, to-10)
pobj(to-10, Camp-11)
cc(Goes-9, and-12)
nsubj(Goes-14, Ernest-13)
conj(Goes-9, Goes-14)
prep(Goes-14, to-15)
pobj(to-15, Jail-16)
punct(are-5, .-17)

Sentence #8 (17 tokens):
So if you have never seen Ernest P. Worrell its time to go and see him.
[Text=So CharacterOffsetBegin=608 CharacterOffsetEnd=610 PartOfSpeech=RB] [Text=if CharacterOffsetBegin=611 CharacterOffsetEnd=613 PartOfSpeech=IN] [Text=you CharacterOffsetBegin=614 CharacterOffsetEnd=617 PartOfSpeech=PRP] [Text=have CharacterOffsetBegin=618 CharacterOffsetEnd=622 PartOfSpeech=VBP] [Text=never CharacterOffsetBegin=623 CharacterOffsetEnd=628 PartOfSpeech=RB] [Text=seen CharacterOffsetBegin=629 CharacterOffsetEnd=633 PartOfSpeech=VBN] [Text=Ernest CharacterOffsetBegin=634 CharacterOffsetEnd=640 PartOfSpeech=NNP] [Text=P. CharacterOffsetBegin=641 CharacterOffsetEnd=643 PartOfSpeech=NNP] [Text=Worrell CharacterOffsetBegin=644 CharacterOffsetEnd=651 PartOfSpeech=NNP] [Text=its CharacterOffsetBegin=652 CharacterOffsetEnd=655 PartOfSpeech=PRP$] [Text=time CharacterOffsetBegin=656 CharacterOffsetEnd=660 PartOfSpeech=NN] [Text=to CharacterOffsetBegin=661 CharacterOffsetEnd=663 PartOfSpeech=TO] [Text=go CharacterOffsetBegin=664 CharacterOffsetEnd=666 PartOfSpeech=VB] [Text=and CharacterOffsetBegin=667 CharacterOffsetEnd=670 PartOfSpeech=CC] [Text=see CharacterOffsetBegin=671 CharacterOffsetEnd=674 PartOfSpeech=VB] [Text=him CharacterOffsetBegin=675 CharacterOffsetEnd=678 PartOfSpeech=PRP] [Text=. CharacterOffsetBegin=678 CharacterOffsetEnd=679 PartOfSpeech=.] 
root(ROOT-0, So-1)
mark(seen-6, if-2)
nsubj(seen-6, you-3)
aux(seen-6, have-4)
neg(seen-6, never-5)
dep(So-1, seen-6)
nn(Worrell-9, Ernest-7)
nn(Worrell-9, P.-8)
dobj(seen-6, Worrell-9)
poss(time-11, its-10)
dep(Worrell-9, time-11)
aux(go-13, to-12)
infmod(time-11, go-13)
cc(go-13, and-14)
conj(go-13, see-15)
dobj(see-15, him-16)
punct(So-1, .-17)

Sentence #9 (13 tokens):
You will find him quite satisfactory "no what I mean"?
[Text=You CharacterOffsetBegin=684 CharacterOffsetEnd=687 PartOfSpeech=PRP] [Text=will CharacterOffsetBegin=688 CharacterOffsetEnd=692 PartOfSpeech=MD] [Text=find CharacterOffsetBegin=693 CharacterOffsetEnd=697 PartOfSpeech=VB] [Text=him CharacterOffsetBegin=698 CharacterOffsetEnd=701 PartOfSpeech=PRP] [Text=quite CharacterOffsetBegin=702 CharacterOffsetEnd=707 PartOfSpeech=RB] [Text=satisfactory CharacterOffsetBegin=708 CharacterOffsetEnd=720 PartOfSpeech=JJ] [Text=`` CharacterOffsetBegin=721 CharacterOffsetEnd=722 PartOfSpeech=``] [Text=no CharacterOffsetBegin=722 CharacterOffsetEnd=724 PartOfSpeech=DT] [Text=what CharacterOffsetBegin=725 CharacterOffsetEnd=729 PartOfSpeech=WP] [Text=I CharacterOffsetBegin=730 CharacterOffsetEnd=731 PartOfSpeech=PRP] [Text=mean CharacterOffsetBegin=732 CharacterOffsetEnd=736 PartOfSpeech=VBP] [Text='' CharacterOffsetBegin=736 CharacterOffsetEnd=737 PartOfSpeech=''] [Text=? CharacterOffsetBegin=737 CharacterOffsetEnd=738 PartOfSpeech=.] 
root(ROOT-0, find-3)
nsubj(find-3, You-1)
aux(find-3, will-2)
dobj(find-3, him-4)
advmod(satisfactory-6, quite-5)
amod(him-4, satisfactory-6)
punct(satisfactory-6, ``-7)
dep(mean-11, no-8)
dobj(mean-11, what-9)
nsubj(mean-11, I-10)
dep(satisfactory-6, mean-11)
punct(satisfactory-6, ''-12)
punct(find-3, ?-13)

